eth_tester/backends/pyevm/serializers.py

```diff
@@ -93,7 +99,7 @@ def serialize_transaction_receipt(block, receipts, transaction, transaction_inde
     else:
         origin_gas = receipt.gas_used - receipts[transaction_index - 1].gas_used
 
-    return {
+    ret = {
```

Comment:
I don't think this ^ change is necessary, so you could revert it. But if you want to keep it, the variable would be easier to understand with a descriptive name -- something like `serialized_receipt`.

Reply:
yeah just `return` is sufficient, I had other code manipulating the dict contents there and then I forgot to revert it back.